br/pkg/task/restore.go

```diff
@@ -122,13 +124,16 @@ type RestoreConfig struct {
 	BatchFlushInterval time.Duration `json:"batch-flush-interval" toml:"batch-flush-interval"`
 	// DdlBatchSize use to define the size of batch ddl to create tables
 	DdlBatchSize uint `json:"ddl-batch-size" toml:"ddl-batch-size"`
+
+	WithPlacementPolicy bool `json:"with-tidb-placement-policy" toml:"with-tidb-placement-policy"`
```

Comment:
According to the issue #32290 , the option should be `--with-tidb-placement-mode=strict/ignore` ?

Reply:
after https://github.com/pingcap/tidb/pull/33007/files#r825267904 changed. this seems normal. I'll fix it.